tests/kernel/kernel_info.cpp

```diff
@@ -44,15 +45,53 @@ TEST_CASE("Test kernel info", "[kernel]") {
 
   /** check program info parameters
    */
-  auto uint32Ret = kernel.get_info<sycl::info::kernel::num_args>();
-  check_return_type<uint32_t>(
-      uint32Ret, "sycl::kernel::get_info<sycl::info::kernel::num_args>()");
+  auto incorrect_num_args_usage = [&kernel] {
+    auto uint32Ret = kernel.get_info<sycl::info::kernel::num_args>();
+  };
+
+  {
+    INFO(
+        "Check that exception with error code \"errc::invalid\" is thrown in "
+        "case of invalid usage of "
```

Comment:
I guess the message should explain why it is invalid.

Reply:
Added.